### ISSUE TITLE
upgrade d3 in arc

### ISSUE THREAD

**Issue by anthonyettinger** [strongloop/strong-arc#1533]
upgrade d3 and all related modules in arc

**Comment by anthonyettinger** [comment]
need to check with @jtary whether this upgrade will break visual composer.

**Comment by jtary** [comment]
Should be fine. If anything breaks, it should be a fairly minor fix.

**Comment by cgole** [comment]
verified

**Comment by cgole** [comment]
@anthonyettinger, should I close this one since we have separate issues for metrics and api analytics and these bugs existed even before the d3 update.

**Comment by seanbrookes** [comment]
as long as we're sure the issues pre-date the library update

On Thu, Sep 17, 2015 at 11:27 PM, Chandrika Gole <notifications@github.com>
wrote:

> @anthonyettinger <https://github.com/anthonyettinger>, should I close
> this one since we have separate issues for metrics and api analytics and
> these bugs existed even before the d3 update.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/strongloop/strong-arc/issues/1533#issuecomment-141356327>
> .
>



-- 
*StrongLoop <http://strongloop.com/>** makes it easy to **develop APIs
<http://strongloop.com/mobile-application-development/loopback/>** in Node,
plus get DevOps capabilities
<http://strongloop.com/node-js-performance/strongops/> like monitoring,
debugging and clustering**.*

**Comment by anthonyettinger** [comment]
Yes, if there are no other problems that arose from upgrading d3. You can close this one.

**Comment by anthonyettinger** [comment]
These have been downgraded to issue with charts Dennis pointed out. Sean has submitted a PR since then to address the issue....so we should try upgrading again when there is sufficient time to test.
